sorc/link_workflow.sh

```diff
@@ -345,10 +345,11 @@ fi
 #--link source code directories
 #------------------------------
 cd "${HOMEgfs}/sorc" || exit 8
-if [[ -d ufs_model.fd ]]; then
-  [[ -d upp.fd ]] && rm -rf upp.fd
-  ${LINK} ufs_model.fd/FV3/upp upp.fd
-fi
+# TODO: Commenting out until UPP is up-to-date with Rocky-8.
```

Comment:
In this comment, do you mean "..until the UPP _**in UWM**_ is up-to-date..."? Wanna avoid confusion with what we're waiting on for later.